.happy/terraform/modules/ecs-stack/main.tf

```diff
@@ -107,6 +107,31 @@ module frontend_service {
   wait_for_steady_state = local.wait_for_steady_state
 }
 
+resource "aws_secretsmanager_secret" "wmg_config" {
+   name = "/corpora/backend/${local.custom_stack_name}/wmg_config"
+}
+  
+variable "wmg_secret_config" {
+  default = {
+    bucket = "cellxgene-wmg-${custom_stack_name}"
```

Comment:
Let's not add this line, since we're removing it entirely in https://github.com/chanzuckerberg/single-cell-data-portal/pull/2677.  If we remove it here, we will want to ensure this PRs is deployed after #2677. Alternately, keep it here for now, but add a new issue to remove it later.